js-exercises/sum-fibs/sumFibs.js

```diff
@@ -0,0 +1,59 @@
+const isNumber = (arg) => typeof arg === 'number';
+
+const isOddNumber = (arg) => isNumber(arg) && (arg % 2 !== 0);
+
+const isPositiveNumber = (arg) => isNumber(arg) && (arg >= 0);
```

Comment:
0 is neither positive not negative. https://math.stackexchange.com/questions/26705/is-zero-positive-or-negative

Reply:
right, code is updated with 3dd02cd49ac897ea7001e3ebeb1253592843577a